experiments/functional/cstor-volume-migration/test.yml

```diff
@@ -0,0 +1,433 @@
+---
+
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+
+   - block:
+
+      ## RECORD START-OF-TEST IN LITMUS RESULT CR
+       - include_tasks: /utils/fcm/update_litmus_result_resource.yml
+         vars:
+           status: 'SOT'
+
+       - name: Get the number of nodes in the cluster
```

Comment:
Get the node names in the cluster

Reply:
@mittachaitu  removed this task as we don't used it anywhere